modules/gnome/nixos.nix

```
@@ -5,12 +5,9 @@ let
 
 in {
   options.stylix.targets.gnome.enable =
-    lib.mkOption {
-      description = "Whether to style GNOME and GDM";
-      type = lib.types.bool;
-      default = config.stylix.autoEnable
-             && config.services.xserver.desktopManager.gnome.enable;
-    };
+    config.lib.stylix.mkEnableTarget
+    "GNOME"
```

```suggestion
    "GNOME and GDM"
```

IMO, this should still specify that it themes GDM (the login screen) in addition to GNOME Shell.

---

I accidentally deleted everything after the first word in the description...